src/couch/src/couch_httpd_auth.erl

```diff
@@ -227,22 +227,27 @@ jwt_authentication_handler(Req) ->
             RequiredClaims = get_configured_claims(),
             case jwtf:decode(?l2b(Jwt), [alg | RequiredClaims], fun jwtf_keystore:get/2) of
                 {ok, {Claims}} ->
+                    couch_log:info("Claims: ~p", [Claims]),
```

Comment:
debugging leftover?

Reply:
for the moment yes, i left them in for the draft, but they are not in the final pr...